dev-util/lxqt-build-tools/files/cpp17-standard.patch

```diff
@@ -0,0 +1,27 @@
+From 4a3a7038b1927dc43942a930db103023efdcbe2c Mon Sep 17 00:00:00 2001
```

Comment:
Please include any relevant references, including the patch source, and any relevant Gentoo bugs at the top of the patch.

Reply:
done